gensim/models/lda_dispatcher.py

```diff
@@ -15,14 +15,21 @@
 
 
 from __future__ import with_statement
-import os, sys, logging, threading, time
+import argparse
```

Comment:
This is py2.7 only. @tmylk I don't think we can drop support for py2.6 yet... is this import safe? If it's just on importing `lda_dispatcher.py`, it's probably fine... but we don't want py2.7+ imports in core gensim (`import gensim`).